tsl/test/sql/continuous_aggs_policy.sql

```diff
@@ -153,6 +158,28 @@ SELECT * FROM mat_smallint;
 SELECT add_continuous_aggregate_policy('mat_smallint', 15::smallint, 10::smallint, '1h'::interval, if_not_exists=>true);
 \set ON_ERROR_STOP 1
 
+--bigint table
+CREATE TABLE bigint_tab (a bigint);
+SELECT table_name FROM create_hypertable('bigint_tab', 'a', chunk_time_interval=> 10);
+CREATE OR REPLACE FUNCTION integer_now_bigint_tab() returns bigint LANGUAGE SQL STABLE as $$ SELECT 20::bigint $$;
+SELECT set_integer_now_func('bigint_tab', 'integer_now_bigint_tab');
+
+CREATE MATERIALIZED VIEW mat_bigint( a, countb )
+WITH (timescaledb.continuous, timescaledb.materialized_only=true)
+as
+SELECT time_bucket( BIGINT '1', a) , count(*)
+FROM bigint_tab
+GROUP BY 1 WITH NO DATA;
+\set ON_ERROR_STOP 0
+SELECT add_continuous_aggregate_policy('mat_bigint', 5::bigint, 10::bigint , '1 h'::interval) ;
+\set ON_ERROR_STOP 1
+SELECT add_continuous_aggregate_policy('mat_bigint', 15::bigint, 0::bigint , '1 h'::interval) as job_id \gset
+INSERT INTO bigint_tab VALUES(5);
+INSERT INTO bigint_tab VALUES(10);
+INSERT INTO bigint_tab VALUES(20);
+CALL run_job(:job_id);
+SELECT * FROM mat_bigint;
+
```

Comment:
can you add a check with '10month' start and '299 day' end and show its accepted but throws runtime error

Reply:
also can you add test with '1000000 year' which would always overflow timestamp and show this is handled properly

Reply:
added